wagtail/admin/tests/test_userbar.py

```diff
@@ -52,27 +51,6 @@ def test_userbar_tag(self):
 
         self.assertIn("<!-- Wagtail user bar embed code -->", content)
 
-    def test_userbar_tag_revision(self):
```

Comment:
Please double-check that removing this test entirely is indeed the right thing to do. I _think_ it was just intending to test the configuration that the now-removed "preview for moderation" view sets up, but it's possible that the `self.assertNumQueries(7)` is doing something important that I didn't pick up on...

Reply:
It was added in https://github.com/wagtail/wagtail/pull/9061 so yep I think it's right, since we no longer have the moderation items